manifests/4.4/cluster-kube-descheduler-operator.v4.4.0.clusterserviceversion.yaml

```diff
@@ -39,100 +39,34 @@ spec:
       name: kubedeschedulers.operator.openshift.io
       version: v1beta1
   description: |
-      # cluster-kube-descheduler-operator
-      An operator to run descheduler on OpenShift
+      The Descheduler Operator provides the ability to evict a running pod so that the pod can be rescheduled onto a more suitable node.
 
-      To deploy the operator:
+      There are several situations where descheduling can benefit your cluster:
 
-      ```
-      oc create -f manifests/.
-      ```
+      * Nodes are underutilized or overutilized.
+      * Pod and node affinity requirements, such as taints or labels, have changed and the original scheduling decisions are no longer appropriate for certain nodes.
+      * Node failure requires pods to be moved.
+      * New nodes are added to clusters.
 
-      Replace `oc` with `kubectl` in case you want descheduler to run with kubernetes. All the required components are created in `openshift-kube-descheduler-operator` namespace.
+      ## Descheduler Strategies
 
-      ## Descheduler strategies
+      Once the operator is installed, you can configure one or more strategies to identify pods to evict. The scheduler will schedule the replacement of the evicted pods.
 
-      The Descheduler operator attempts to simplify the descheduler strategy names from their [upstream names](https://github.com/kubernetes-sigs/descheduler/#policy-and-strategies). Thus when set on the operator, these strategy names map to:
+      The following strategies are supported:
 
-      | Operator param | Descheduler strategy |
-      | ---- | ---- |
-      | `duplicates` | `RemoveDuplicates` |
-      | `interpodantiaffinity` | `RemovePodsViolatingInterPodAntiAffinity` |
-      | `lownodeutilization` | `LowNodeUtilization` |
-      | `nodeaffinity` | `RemovePodsViolatingNodeAffinity` |
-      | `nodetaints` | `RemovePodsViolatingNodeTaints` |
+      * `LowNodeUtilization` - This strategy finds nodes that are underutilized and evicts pods, if possible, from other nodes in the hope that recreation of evicted pods will be scheduled on these underutilized nodes.
+      * `RemoveDuplicates` - This strategy ensures that there is only one pod associated with a ReplicaSet, ReplicationController, Deployment, or Job running on same node. If there are more, then those duplicate pods are evicted for better spreading of pods in a cluster.
+      * `RemovePodsViolatingInterPodAntiAffinity` - This strategy ensures that pods violating inter-pod anti-affinity are removed from nodes.
+      * `RemovePodsViolatingNodeAffinity` - This strategy ensures that pods violating node affinity are removed from nodes.
+      * `RemovePodsViolatingNodeTaints` - This strategy ensures that pods violating `NoSchedule` taints on nodes are removed.
 
-      ## Sample CR
```

Comment:
Just one nit. Let's put `These strategies are documented in detail in the [descheduler README](https://github.com/kubernetes-sigs/descheduler/#policy-and-strategies).` right at the end of `## Descheduler Strategies` section.